### DISCUSSION TITLE
Terminal feature

### DISCUSSION THREAD

**Discussion by marcos1500** [Ylianst/MeshCentral#5606] [General]
Hello, everyone.

I would like to know how I can implement custom commands in the dropdown menu on the terminal screen.

To have some commands already pre-configured, for example, 'net stop spooler,' among others. Does anyone know where I can configure these commands to become default as well?

Thank you, everyone.
![2023-12-04 09 04 46](https://github.com/Ylianst/MeshCentral/assets/24254685/b9464355-42a5-4de9-b9e1-7090cd7a46e3)

**Comment by si458** [comment]
good feature request!
this is already implemented in the desktop tab, by right clicking the 'type' button and adding what to type/send

**Reply by marcos1500** [reply]
Simon Smith, thank you; however, I was already aware of this feature on the desktop.
I truly want to implement these custom commands on the terminal screen without needing to interact with the user interface.
If possible, please let me know where the code for the mentioned field is located, so I can implement it there directly.

I'm not an expert in programming, hahaha, but I manage my way around. I enjoy exploring things.
Even so, thank you very much for your response.

**Reply by si458** [reply]
the file you need to edit is `default.handlebars` inside `views`, in there you put a button and code to do stuff
